packages/devtools_app/lib/src/shared/connected_app.dart

```diff
@@ -36,62 +36,62 @@ class ConnectedApp {
   Future<bool> get isFlutterApp async => _isFlutterApp ??=
       await serviceManager.libraryUriAvailable(flutterLibraryUri);
 
-  bool get isFlutterAppNow {
+  bool? get isFlutterAppNow {
     assert(_isFlutterApp != null);
-    return _isFlutterApp;
+    return _isFlutterApp == true;
   }
 
-  bool _isFlutterApp;
+  bool? _isFlutterApp;
 
-  FlutterVersion get flutterVersionNow {
+  FlutterVersion? get flutterVersionNow {
     return isFlutterNativeAppNow ? _flutterVersion : null;
   }
 
-  FlutterVersion _flutterVersion;
+  FlutterVersion? _flutterVersion;
 
-  final _flutterVersionCompleter = Completer<FlutterVersion>();
+  final _flutterVersionCompleter = Completer<FlutterVersion?>();
 
   static const _flutterVersionTimeout = Duration(seconds: 3);
 
-  Future<bool> get isProfileBuild async {
+  Future<bool?> get isProfileBuild async {
```

Comment:
this, isProfileBuildNow, and isDartWebAppNow can all be non-nullable. Same comment about their respective private fields with the question as to whether we can make them late instead of nullable

Reply:
fixed

Reply:
reverted, because they are overriden too